frontend/src/main/scala/bloop/engine/tasks/TestTask.scala

```diff
@@ -337,7 +337,7 @@ object TestTask {
         taskDefs.map {
           case TaskDefWithFramework(taskDef, _) =>
             selectedTests.get(taskDef.fullyQualifiedName()) match {
-              case None =>
+              case None | Some(Nil) =>
```

Comment:
This fixes the DebugProtocolSuite tests (previously it was
```scala
 selectedTests.get(taskDef.fullyQualifiedName()).getOrElse(Nil) match {
              case Nil =>
              case selectedTests =>
```
and I think the Some(Nil) got lost in the refactor

Reply:
Ach! Good catch!